### ISSUE TITLE
[question] What datasets are used in the training of stt eu model?

### ISSUE THREAD

**Issue by SatenHarutyunyan** [NVIDIA/NeMo#9614]
Hi. I'm interested what datasets did you use for this model?
https://catalog.ngc.nvidia.com/orgs/nvidia/teams/nemo/models/stt_multilingual_fastconformer_hybrid_large_pc_blend_eu

**Comment by SatenHarutyunyan** [comment]
Also, did you use nemo text normalization for text preprocessing? If yes, did you use only it?

**Comment by bmwshop** [comment]
> Hi. I'm interested what datasets did you use for this model? https://catalog.ngc.nvidia.com/orgs/nvidia/teams/nemo/models/stt_multilingual_fastconformer_hybrid_large_pc_blend_eu

We used a combination of the train sets used to train English, Spanish, French and German monolingual models.

**Comment by bmwshop** [comment]
> Also, did you use nemo text normalization for text preprocessing? If yes, did you use only it?

i believe so.

**Comment by SatenHarutyunyan** [comment]
thank you for answering

Its interesting, because we tried to resume the training (with NeMo) on the same dataset and load from pretrained nemo, and the training improves on validation set, but deteriorates on fleurs test set. Thats why we thought maybe you used other cleaning or processing of the text label.

**Comment by erastorgueva-nv** [comment]
Hello, I see this is essentially a duplicate of https://github.com/NVIDIA/NeMo/issues/10122, which I have answered. I hope we have been able to help you. Can I mark both these issues closed?

**Comment by SatenHarutyunyan** [comment]
Yes, thank you for answering
